admin_guide/install/first_steps.adoc

```diff
@@ -48,6 +48,11 @@ deployed in your OpenShift installation.
 link:../../architecture/additional_concepts/authorization.html#roles[*cluster-admin*
 privileges], because they operate on the default *openshift*
 link:../../architecture/core_concepts/projects_and_users.html#projects[project].
+- You must have cloned the link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files[repository] that contains the supported imagestreams:
++
+----
+$ git clone https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files/
```

Comment:
The clone operation must be in regards to repository, not a directory inside:
```
git clone https://github.com/openshift/openshift-ansible/
```
Only after you can link to a directory inside the cloned repository.